### PR TITLE
using devtools to get console log

### PR DESCRIPTION
**Pull request by newtewt**
# Description

@dianabarsan  Draft PR for devtools for logging. The output is shown below. There are duplicates that I'm unsure exactly why those are coming in.  The `Runtime.consoleAPICalled` event has so many entries it is insane. I'll tidy up a bit before a full PR but wanted your thoughts.  

```
[error]: network Failed to load resource: the server responded with a status of 404 (Object Not Found) url: http://localhost:4988/medic-user-admin-meta/ at 1628870525281.921
[error]: network Failed to load resource: the server responded with a status of 404 (Object Not Found) url: http://localhost:4988/medic-user-admin-meta/ at 1628870525281.921
[error]: network Failed to load resource: the server responded with a status of 404 (Object Not Found) url: http://localhost:4988/medic/partners? at 1628870525289.837
[error]: network Failed to load resource: the server responded with a status of 404 (Object Not Found) url: http://localhost:4988/medic/partners? at 1628870525289.837
[error]: network Failed to load resource: the server responded with a status of 404 (Object Not Found) url: http://localhost:4988/medic/privacy-policies? at 1628870525730.214
[error]: network Failed to load resource: the server responded with a status of 404 (Object Not Found) url: http://localhost:4988/medic/privacy-policies? at 1628870525730.214
~~~~~~~~~~~~~ run and end guided tour => should close the tour ~~~~~~~~~~~~~~~~~~~~~~
[error]: network Failed to load resource: the server responded with a status of 404 (Object Not Found) url: http://localhost:4988/medic/partners? at 1628870531431.571
[error]: network Failed to load resource: the server responded with a status of 404 (Object Not Found) url: http://localhost:4988/medic/partners? at 1628870531431.571
[error]: network Failed to load resource: the server responded with a status of 404 (Object Not Found) url: http://localhost:4988/medic/privacy-policies? at 1628870531587.7222
[error]: network Failed to load resource: the server responded with a status of 404 (Object Not Found) url: http://localhost:4988/medic/privacy-policies? at 1628870531587.7222
~~~~~~~~~~~~~ run guided tour and change tab => should close the tour ~~~~~~~~~~~~~~~~~~~~~~
[error]: network Failed to load resource: the server responded with a status of 404 (Object Not Found) url: http://localhost:4988/medic/partners? at 1628870542658.469
[error]: network Failed to load resource: the server responded with a status of 404 (Object Not Found) url: http://localhost:4988/medic/partners? at 1628870542658.469
[error]: network Failed to load resource: the server responded with a status of 404 (Object Not Found) url: http://localhost:4988/medic/privacy-policies? at 1628870542832.7
[error]: network Failed to load resource: the server responded with a status of 404 (Object Not Found) url: http://localhost:4988/medic/privacy-policies? at 1628870542832.7
~~~~~~~~~~~~~ Create new health center ~~~~~~~~~~~~~~~~~~~~~~
log Event: [{"type":"string","value":"No active pages found. Make sure your form has at least one relevant input."}]
log Event: [{"type":"string","value":"No active pages found. Make sure your form has at least one relevant input."}]
log Event: [{"type":"string","value":"No active pages found. Make sure your form has at least one relevant input."}]
log Event: [{"type":"string","value":"No active pages found. Make sure your form has at least one relevant input."}]
log Event: [{"type":"string","value":"No active pages found. Make sure your form has at least one relevant input."}]
log Event: [{"type":"string","value":"No active pages found. Make sure your form has at least one relevant input."}]
log Event: [{"type":"string","value":"Attempt to run transitions without initialization"}]
log Event: [{"type":"string","value":"Attempt to run transitions without initialization"}]
~~~~~~~~~~~~~ Create new area ~~~~~~~~~~~~~~~~~~~~~~
log Event: [{"type":"string","value":"No active pages found. Make sure your form has at least one relevant input."}]
log Event: [{"type":"string","value":"No active pages found. Make sure your form has at least one relevant input."}]
log Event: [{"type":"string","value":"No active pages found. Make sure your form has at least one relevant input."}]
log Event: [{"type":"string","value":"No active pages found. Make sure your form has at least one relevant input."}]
log Event: [{"type":"string","value":"No active pages found. Make sure your form has at least one relevant input."}]
log Event: [{"type":"string","value":"No active pages found. Make sure your form has at least one relevant input."}]
log Event: [{"type":"string","value":"Attempt to run transitions without initialization"}]
log Event: [{"type":"string","value":"Attempt to run transitions without initialization"}]
~~~~~~~~~~~~~ Create new household ~~~~~~~~~~~~~~~~~~~~~~
log Event: [{"type":"string","value":"No active pages found. Make sure your form has at least one relevant input."}]
log Event: [{"type":"string","value":"No active pages found. Make sure your form has at least one relevant input."}]
log Event: [{"type":"string","value":"No active pages found. Make sure your form has at least one relevant input."}]
log Event: [{"type":"string","value":"No active pages found. Make sure your form has at least one relevant input."}]
log Event: [{"type":"string","value":"No active pages found. Make sure your form has at least one relevant input."}]
log Event: [{"type":"string","value":"No active pages found. Make sure your form has at least one relevant input."}]
log Event: [{"type":"string","value":"Attempt to run transitions without initialization"}]
log Event: [{"type":"string","value":"Attempt to run transitions without initialization"}]
~~~~~~~~~~~~~ Create new person ~~~~~~~~~~~~~~~~~~~~~~
log Event: [{"type":"string","value":"Attempt to run transitions without initialization"}]
log Event: [{"type":"string","value":"Attempt to run transitions without initialization"}]
log Event: [{"type":"string","value":"Error selecting contact"},{"type":"object","subtype":"error","className":"Error","description":"Error: Document not found: 176b07b3-a090-49f8-87ef-96890d8ba5c6\n    at new LineageError (http://localhost:4988/main.js:333038:28)\n    at http://localhost:4988/main.js:332934:27\n    at ZoneDelegate.invoke (http://localhost:4988/polyfills.js:790:160)\n    at Zone.run (http://localhost:4988/polyfills.js:516:37)\n    at http://localhost:4988/polyfills.js:1936:28\n    at ZoneDelegate.invokeTask (http://localhost:4988/polyfills.js:823:173)\n    at Zone.runTask (http://localhost:4988/polyfills.js:573:39)\n    at drainMicroTaskQueue (http://localhost:4988/polyfills.js:1039:25)","objectId":"8170477764220469685.10.1122","preview":{"type":"object","subtype":"error","description":"Error: Document not found: 176b07b3-a090-49f8-87ef-96890d8ba5c6\n    at new LineageError (http://localhost:4988/main.js:333038:28)\n    at http://localhost:4988/main.js:332934:27\n    at ZoneDelegate.invoke (http://localhost:4988/polyfills.js:790:160)\n    at Zone.run (http://localhost:4988/polyfills.js:516:37)\n    at http://localhost:4988/polyfills.js:1936:28\n    at ZoneDelegate.invokeTask (http://localhost:4988/polyfills.js:823:173)\n    at Zone.runTask (http://localhost:4988/polyfills.js:573:39)\n    at drainMicroTaskQueue (http://localhost:4988/polyfills.js:1039:25)","overflow":false,"properties":[{"name":"code","type":"number","value":"404"},{"name":"stack","type":"string","value":"Error: Document not found: 176b07b3-a090-49f8-87ef…ueue (http://localhost:4988/polyfills.js:1039:25)"},{"name":"message","type":"string","value":"Document not found: 176b07b3-a090-49f8-87ef-96890d8ba5c6"}]}}]
log Event: [{"type":"string","value":"Error selecting contact"},{"type":"object","subtype":"error","className":"Error","description":"Error: Document not found: 176b07b3-a090-49f8-87ef-96890d8ba5c6\n    at new LineageError (http://localhost:4988/main.js:333038:28)\n    at http://localhost:4988/main.js:332934:27\n    at ZoneDelegate.invoke (http://localhost:4988/polyfills.js:790:160)\n    at Zone.run (http://localhost:4988/polyfills.js:516:37)\n    at http://localhost:4988/polyfills.js:1936:28\n    at ZoneDelegate.invokeTask (http://localhost:4988/polyfills.js:823:173)\n    at Zone.runTask (http://localhost:4988/polyfills.js:573:39)\n    at drainMicroTaskQueue (http://localhost:4988/polyfills.js:1039:25)","objectId":"8170477764220469685.10.1122","preview":{"type":"object","subtype":"error","description":"Error: Document not found: 176b07b3-a090-49f8-87ef-96890d8ba5c6\n    at new LineageError (http://localhost:4988/main.js:333038:28)\n    at http://localhost:4988/main.js:332934:27\n    at ZoneDelegate.invoke (http://localhost:4988/polyfills.js:790:160)\n    at Zone.run (http://localhost:4988/polyfills.js:516:37)\n    at http://localhost:4988/polyfills.js:1936:28\n    at ZoneDelegate.invokeTask (http://localhost:4988/polyfills.js:823:173)\n    at Zone.runTask (http://localhost:4988/polyfills.js:573:39)\n    at drainMicroTaskQueue (http://localhost:4988/polyfills.js:1039:25)","overflow":false,"properties":[{"name":"code","type":"number","value":"404"},{"name":"stack","type":"string","value":"Error: Document not found: 176b07b3-a090-49f8-87ef…ueue (http://localhost:4988/polyfills.js:1039:25)"},{"name":"message","type":"string","value":"Document not found: 176b07b3-a090-49f8-87ef-96890d8ba5c6"}]}}]
[error]: network Failed to load resource: the server responded with a status of 404 (Object Not Found) url: http://localhost:4988/medic/partners? at 1628870561009.667
[error]: network Failed to load resource: the server responded with a status of 404 (Object Not Found) url: http://localhost:4988/medic/partners? at 1628870561009.667
[error]: network Failed to load resource: the server responded with a status of 404 (Object Not Found) url: http://localhost:4988/medic/privacy-policies? at 1628870561302.784
[error]: network Failed to load resource: the server responded with a status of 404 (Object Not Found) url: http://localhost:4988/medic/privacy-policies? at 1628870561302.784
~~~~~~~~~~~~~ create person under area should only see children ~~~~~~~~~~~~~~~~~~~~~~
log Event: [{"type":"string","value":"Error updating icons"},{"type":"object","subtype":"error","className":"TypeError","description":"TypeError: Failed to fetch","objectId":"-5913679431255272837.10.82","preview":{"type":"object","subtype":"error","description":"TypeError: Failed to fetch","overflow":false,"properties":[{"name":"docId","type":"string","value":"resources"},{"name":"stack","type":"string","value":"TypeError: Failed to fetch"},{"name":"message","type":"string","value":"Failed to fetch"}]}}]
log Event: [{"type":"string","value":"Error updating icons"},{"type":"object","subtype":"error","className":"TypeError","description":"TypeError: Failed to fetch","objectId":"-5913679431255272837.10.82","preview":{"type":"object","subtype":"error","description":"TypeError: Failed to fetch","overflow":false,"properties":[{"name":"docId","type":"string","value":"resources"},{"name":"stack","type":"string","value":"TypeError: Failed to fetch"},{"name":"message","type":"string","value":"Failed to fetch"}]}}]
log Event: [{"type":"string","value":"User must reauthenticate"}]
log Event: [{"type":"string","value":"User must reauthenticate"}]
log Event: [{"type":"string","value":"Error watching for db changes"},{"type":"object","subtype":"error","className":"TypeError","description":"TypeError: Failed to fetch","objectId":"-5913679431255272837.13.8","preview":{"type":"object","subtype":"error","description":"TypeError: Failed to fetch","overflow":false,"properties":[{"name":"stack","type":"string","value":"TypeError: Failed to fetch"},{"name":"message","type":"string","value":"Failed to fetch"}]}}]
log Event: [{"type":"string","value":"Error watching for db changes"},{"type":"object","subtype":"error","className":"TypeError","description":"TypeError: Failed to fetch","objectId":"-5913679431255272837.13.8","preview":{"type":"object","subtype":"error","description":"TypeError: Failed to fetch","overflow":false,"properties":[{"name":"stack","type":"string","value":"TypeError: Failed to fetch"},{"name":"message","type":"string","value":"Failed to fetch"}]}}]
log Event: [{"type":"string","value":"Attempting changes reconnection in 5 seconds"}]
log Event: [{"type":"string","value":"Attempting changes reconnection in 5 seconds"}]
log Event: [{"type":"string","value":"Error watching for db changes"},{"type":"object","subtype":"error","className":"TypeError","description":"TypeError: Failed to fetch","objectId":"-5913679431255272837.13.9","preview":{"type":"object","subtype":"error","description":"TypeError: Failed to fetch","overflow":false,"properties":[{"name":"stack","type":"string","value":"TypeError: Failed to fetch"},{"name":"message","type":"string","value":"Failed to fetch"}]}}]
log Event: [{"type":"string","value":"Error watching for db changes"},{"type":"object","subtype":"error","className":"TypeError","description":"TypeError: Failed to fetch","objectId":"-5913679431255272837.13.9","preview":{"type":"object","subtype":"error","description":"TypeError: Failed to fetch","overflow":false,"properties":[{"name":"stack","type":"string","value":"TypeError: Failed to fetch"},{"name":"message","type":"string","value":"Failed to fetch"}]}}]
log Event: [{"type":"string","value":"Attempting changes reconnection in 5 seconds"}]
log Event: [{"type":"string","value":"Attempting changes reconnection in 5 seconds"}]
[error]: network Failed to load resource: the server responded with a status of 404 (Object Not Found) url: http://localhost:4988/medic/_local/4otUhmGaLbhHZh7XySqWTw%3D%3D? at 1628870571184.552
[error]: network Failed to load resource: the server responded with a status of 404 (Object Not Found) url: http://localhost:4988/medic/_local/4otUhmGaLbhHZh7XySqWTw%3D%3D? at 1628870571184.552
log Event: [{"type":"string","value":"Error fetching read status"},{"type":"object","className":"CustomPouchError","description":"CustomPouchError","objectId":"-5913679431255272837.19.70","preview":{"type":"object","description":"CustomPouchError","overflow":true,"properties":[{"name":"status","type":"number","value":"404"},{"name":"name","type":"string","value":"not_found"},{"name":"message","type":"string","value":"missing"},{"name":"error","type":"boolean","value":"true"},{"name":"reason","type":"string","value":"missing"}]}}]
log Event: [{"type":"string","value":"Error fetching read status"},{"type":"object","className":"CustomPouchError","description":"CustomPouchError","objectId":"-5913679431255272837.19.70","preview":{"type":"object","description":"CustomPouchError","overflow":true,"properties":[{"name":"status","type":"number","value":"404"},{"name":"name","type":"string","value":"not_found"},{"name":"message","type":"string","value":"missing"},{"name":"error","type":"boolean","value":"true"},{"name":"reason","type":"string","value":"missing"}]}}]
~~~~~~~~~~~~~ should show locale selector on login page ~~~~~~~~~~~~~~~~~~~~~~
~~~~~~~~~~~~~ should change locale to French ~~~~~~~~~~~~~~~~~~~~~~
~~~~~~~~~~~~~ should change locale to Spanish ~~~~~~~~~~~~~~~~~~~~~~
~~~~~~~~~~~~~ should show a warning before log out ~~~~~~~~~~~~~~~~~~~~~~
[error]: network Failed to load resource: the server responded with a status of 404 (Object Not Found) url: http://localhost:4988/medic/partners? at 1628870580728.356
[error]: network Failed to load resource: the server responded with a status of 404 (Object Not Found) url: http://localhost:4988/medic/partners? at 1628870580728.356
[error]: network Failed to load resource: the server responded with a status of 404 (Object Not Found) url: http://localhost:4988/medic/privacy-policies? at 1628870580960.264
[error]: network Failed to load resource: the server responded with a status of 404 (Object Not Found) url: http://localhost:4988/medic/privacy-policies? at 1628870580960.264
log Event: [{"type":"string","value":"User must reauthenticate"}]
log Event: [{"type":"string","value":"User must reauthenticate"}]
~~~~~~~~~~~~~ should log in using username and password fields ~~~~~~~~~~~~~~~~~~~~~~
[error]: network Failed to load resource: the server responded with a status of 404 (Object Not Found) url: http://localhost:4988/medic/partners? at 1628870583853.852
[error]: network Failed to load resource: the server responded with a status of 404 (Object Not Found) url: http://localhost:4988/medic/partners? at 1628870583853.852
[error]: network Failed to load resource: the server responded with a status of 404 (Object Not Found) url: http://localhost:4988/medic/privacy-policies? at 1628870584006.448
[error]: network Failed to load resource: the server responded with a status of 404 (Object Not Found) url: http://localhost:4988/medic/privacy-policies? at 1628870584006.448
[error]: network Failed to load resource: the server responded with a status of 401 (Unauthorized) url: http://localhost:4988/medic/resources/icon-healthcare-warning%402x.png?rev=2-829966961db41474c748a7af216a22ed at 1628870584353.994
[error]: network Failed to load resource: the server responded with a status of 401 (Unauthorized) url: http://localhost:4988/medic/resources/icon-healthcare-warning%402x.png?rev=2-829966961db41474c748a7af216a22ed at 1628870584353.994
[error]: network Failed to load resource: the server responded with a status of 401 (Unauthorized) url: http://localhost:4988/medic/resources/icon-messages-off%402x.png?rev=2-829966961db41474c748a7af216a22ed at 1628870584386.486
[error]: network Failed to load resource: the server responded with a status of 401 (Unauthorized) url: http://localhost:4988/medic/resources/icon-messages-off%402x.png?rev=2-829966961db41474c748a7af216a22ed at 1628870584386.486
[error]: network Failed to load resource: the server responded with a status of 401 (Unauthorized) url: http://localhost:4988/medic/resources/icon-messages-on%402x.png?rev=2-829966961db41474c748a7af216a22ed at 1628870584398.895
[error]: network Failed to load resource: the server responded with a status of 401 (Unauthorized) url: http://localhost:4988/medic/resources/icon-messages-on%402x.png?rev=2-829966961db41474c748a7af216a22ed at 1628870584398.895
[error]: network Failed to load resource: the server responded with a status of 401 (Unauthorized) url: http://localhost:4988/medic/resources/icon-people-CHW-crop%402x.png?rev=2-829966961db41474c748a7af216a22ed at 1628870584433.5461
[error]: network Failed to load resource: the server responded with a status of 401 (Unauthorized) url: http://localhost:4988/medic/resources/icon-people-CHW-crop%402x.png?rev=2-829966961db41474c748a7af216a22ed at 1628870584433.5461
log Event: [{"type":"string","value":"User must reauthenticate"}]
log Event: [{"type":"string","value":"User must reauthenticate"}]
[error]: network Failed to load resource: the server responded with a status of 404 (Object Not Found) url: http://localhost:4988/medic/partners? at 1628870590747.37
[error]: network Failed to load resource: the server responded with a status of 404 (Object Not Found) url: http://localhost:4988/medic/partners? at 1628870590747.37
[error]: network Failed to load resource: the server responded with a status of 404 (Object Not Found) url: http://localhost:4988/medic/privacy-policies? at 1628870590911.4001
[error]: network Failed to load resource: the server responded with a status of 404 (Object Not Found) url: http://localhost:4988/medic/privacy-policies? at 1628870590911.4001
~~~~~~~~~~~~~ create case ID ~~~~~~~~~~~~~~~~~~~~~~
[error]: network Failed to load resource: the server responded with a status of 404 (Object Not Found) url: http://localhost:4988/medic/partners? at 1628870600237.4922
[error]: network Failed to load resource: the server responded with a status of 404 (Object Not Found) url: http://localhost:4988/medic/partners? at 1628870600237.4922
[error]: network Failed to load resource: the server responded with a status of 404 (Object Not Found) url: http://localhost:4988/medic/privacy-policies? at 1628870600476.871
[error]: network Failed to load resource: the server responded with a status of 404 (Object Not Found) url: http://localhost:4988/medic/privacy-policies? at 1628870600476.871
~~~~~~~~~~~~~ search by NON empty string should display results with contains match and clears search ~~~~~~~~~~~~~~~~~~~~~~
[error]: network Failed to load resource: the server responded with a status of 404 (Object Not Found) url: http://localhost:4988/medic/partners? at 1628870610526.384
[error]: network Failed to load resource: the server responded with a status of 404 (Object Not Found) url: http://localhost:4988/medic/partners? at 1628870610526.384
[error]: network Failed to load resource: the server responded with a status of 404 (Object Not Found) url: http://localhost:4988/medic/privacy-policies? at 1628870610812.689
[error]: network Failed to load resource: the server responded with a status of 404 (Object Not Found) url: http://localhost:4988/medic/privacy-policies? at 1628870610812.689
~~~~~~~~~~~~~ search by NON empty string should display results with contains match and then clears ~~~~~~~~~~~~~~~~~~~~~~
[error]: network Failed to load resource: the server responded with a status of 404 (Object Not Found) url: http://localhost:4988/medic/partners? at 1628870620836.538
[error]: network Failed to load resource: the server responded with a status of 404 (Object Not Found) url: http://localhost:4988/medic/partners? at 1628870620836.538
[error]: network Failed to load resource: the server responded with a status of 404 (Object Not Found) url: http://localhost:4988/medic/privacy-policies? at 1628870621032.625
[error]: network Failed to load resource: the server responded with a status of 404 (Object Not Found) url: http://localhost:4988/medic/privacy-policies? at 1628870621032.625
~~~~~~~~~~~~~ should display plurals correctly ~~~~~~~~~~~~~~~~~~~~~~
~~~~~~~~~~~~~ should work with botched translations ~~~~~~~~~~~~~~~~~~~~~~
log Event: [{"type":"string","value":"messageformat compile error"},{"type":"object","className":"peg$SyntaxError","description":"peg$SyntaxError","objectId":"857604454322489260.10.330","preview":{"type":"object","description":"peg$SyntaxError","overflow":true,"properties":[{"name":"message","type":"string","value":"Expected \",\" or \"}\" but end of input found."},{"name":"expected","type":"object","value":"Array(4)","subtype":"array"},{"name":"found","type":"object","value":"null","subtype":"null"},{"name":"location","type":"object","value":"Object"},{"name":"name","type":"string","value":"SyntaxError"}]}}]
log Event: [{"type":"string","value":"messageformat compile error"},{"type":"object","className":"peg$SyntaxError","description":"peg$SyntaxError","objectId":"857604454322489260.10.330","preview":{"type":"object","description":"peg$SyntaxError","overflow":true,"properties":[{"name":"message","type":"string","value":"Expected \",\" or \"}\" but end of input found."},{"name":"expected","type":"object","value":"Array(4)","subtype":"array"},{"name":"found","type":"object","value":"null","subtype":"null"},{"name":"location","type":"object","value":"Object"},{"name":"name","type":"string","value":"SyntaxError"}]}}]
log Event: [{"type":"string","value":"Error while interpolating"},{"type":"string","value":"Messages {thing}"}]
log Event: [{"type":"string","value":"Error while interpolating"},{"type":"string","value":"Messages {thing}"}]
log Event: [{"type":"string","value":"Error while interpolating"},{"type":"string","value":"Messages {thing}"}]
log Event: [{"type":"string","value":"Error while interpolating"},{"type":"string","value":"Messages {thing}"}]
```

### CODE REVIEW

File: tests/wdio.conf.js [hunk 1]
```diff
@@ -227,6 +228,24 @@ const baseConfig = {
    */
   before: async function () {
     await browser.url('/');
+    await browser.cdp('Log', 'enable');
+    await browser.cdp('Runtime', 'enable');
+    browser.on('Runtime.consoleAPICalled', (data) => {
+      if (data &&(data.type === 'error' || data.type ==='warning' )) {
+        const logEntry = `log Event: ${JSON.stringify(data.args)}\n`;
```
Comment (dianabarsan) [inline tests/wdio.conf.js:235]: Please add the level somewhere here.

File: tests/wdio.conf.js [hunk 2]
```diff
@@ -227,6 +228,24 @@ const baseConfig = {
...
+        console.log(`~~~~~ log event ~~~~~`);
+        console.log(logEntry);
+        console.log(`~~~~~ log event end ~~~~~`);
+        fs.appendFileSync(browserLogPath, logEntry);
```
Comment (dianabarsan) [inline tests/wdio.conf.js:239]: Maybe the duplicates probably happen because the same log event fires `Runtime.consoleAPICalled` AND `Log.entryAdded` ?

Comment (newtewt) [reply tests/wdio.conf.js:239]: Sadly it was occurring when before I had implemented the `Runtime.consoleAPICalled` .

Comment (dianabarsan) [reply tests/wdio.conf.js:239]: And is the event indeed fired twice? You get double logs in the test console as well?

Comment (newtewt) [reply tests/wdio.conf.js:239]: yeah it's doubled in both. I'm not sure why.  I'll dig a bit more.

Comment (newtewt) [reply tests/wdio.conf.js:239]: A bug?

https://github.com/webdriverio/webdriverio/issues/6347

**Comment by newtewt** [comment]
@dianabarsan  what do you think about going forward with this change with webdriverio/webdriverio#6347?

**Comment by garethbowen** [comment]
@dianabarsan This looks useful. Other than the conflicting files, can you remember what else needs to be done to get this merged?

**Comment by dianabarsan** [comment]
I believe addressing the duplicate entries was the issue left to be resolved.

**Comment by garethbowen** [comment]
@dianabarsan Rebased, and made a cheap hack to dedupe the logs to work around the known issue. Review please!

**Review by dianabarsan — APPROVED** [review]
Awesome!

**Pull request merged**